docs/swagger_v3/names.spec.yaml

```diff
@@ -1,4 +1,30 @@
 schemas:
+  NameTx:
+  properties:
```

Comment:
This doesn't appear to be nested inside NameTx. Also, what is a NameTx? is it claim, update, revoke?

Reply:
This is so i can resue the structure inside of the latter schemas